docs/content/projects/cat.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "OSCAL CAT"
+description: "OSCAL Catalog Authoring Tool (CAT) - import, edit, and export OSCAL Catalogs from the browser"
+heading: "CAT: OSCAL Catalog Authoring Tool"
+toc:
+  enabled: true
+usabanner: true
+---
+
+## What is CAT?
+
+The OSCAL Catalog Authoring Tool (CAT) enables users to create, scope, tailor, control catalogs in [the OSCAL JSON format](https://pages.nist.gov/OSCAL/reference/latest/catalog/json-reference/).
```

Comment:
- no comma is needed after 'tailor'. 
- I spoke with Dmitry and CAT is not modifying catalogs. It can only import an EXISTING OSCAL catalog and is able to select controls, tailor them and export an OSCAL profile (it is not resolving it). 
- So it is actually a Profile Authoring Tool. The imported Catalog cannot be modified/corrected and saved/exported as of today. No new controls can be added.

Reply:
Comma typo updated, the more substantive feature listing will be addressed elsewhere in amended commits.